cli/tauri.js/bin/tauri.js

```diff
@@ -2,9 +2,11 @@
 
 const cmds = ['init', 'help', 'icon', 'info', 'deps']
 const rustCliCmds = ['dev', 'build']
-
+const fs = require('fs');
 const cmd = process.argv[2]
-
+const chalk = require('chalk');
+const package = require('../package.json');
+const figlet = require('figlet');
```

Comment:
This isn't specified in the `package.json`.

Reply:
So I change the `package.json` as well?